src/opcodes/tests/test_opcodes.cairo

```diff
@@ -691,3 +691,19 @@ fn test_op_lessthan_equal() {
     let expected_stack = array![""];
     assert_eq!(dstack, expected_stack.span(), "Stack is not equal to expected");
 }
+
+#[test]
```

Comment:
Then in this test, we will want to check that an error was thrown using the `should_panic` declaration: https://book.cairo-lang.org/ch10-01-how-to-write-tests.html?highlight=expect#checking-for-panics-with-should_panic